mzbench/scenarios/1to1.bdl

```diff
@@ -0,0 +1,56 @@
+#!benchDL
+
+#######
+# 1_to_1 Scenario:
+# 1000 clients subscribe to an exclusive topic: "channels/{channel_id}/messages"
```

Comment:
I would prefer 1000 publisher-only and 1000 subscriber-only. Also - please avoid "loopbacks" - i.e. sending message to yourself - this is unrealistic scenario: why would you send a message to yourself when you have it already.